include/mbedtls/x509_crt.h

```diff
@@ -156,13 +156,17 @@ mbedtls_x509_subject_alternative_name;
  * Security profile for certificate verification.
  *
  * All lists are bitfields, built by ORing flags from MBEDTLS_X509_ID_FLAG().
+ *
+ * The fields of this structure are part of the public API and can be
+ * manipulated directly by applications. Future versions of the library may
+ * add extra fields or reorder existing fields.
```

Comment:
Hmm, we may want to change field types either. Not very likely, but suppose we decide to add a 33rd curve before Mbed TLS 4.0 comes out...